src/search.js

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+
+//MAAK EEN VARIABELE AAN MET DE NAAM USERINPUT
+//HAAL DE GEGEVENS OP VAN USER-INPUT
+const userInput = document.getElementById('user-input');
+//MAAK EEN VARIABELE AAN MET DE NAAM SUBMITBTN
+//HAAL DE GEGEVENS OP VAN SUBMITBUTTON
+const submitBtn = document.getElementById('submitButton');
+//ZET EEN EVENTLISTENER OP CLICK VAN SUBMITBUTTON EN ACTIVEER FUNCTIE GETUSERINPUT
+submitBtn.addEventListener('click', getUserInput);
+
+//MAAK EEN FUNCTIE VOOR HET OPHALEN VAN EEN LAND
+function getUserInput(e) {
+    e.preventDefault();
+    getData(userInput.value);
+    userInput.value = '';
+}
+
+
+//MAAK EEN ASYNCHRONE FUNCTIE VOOR HET OPHALEN VAN DATA
+async function getData(searchName){
+    userInput.innerHTML = ``;
+    let searchResult;
+    try{
+            //MAAK EEN VARIABELE RESULT VOOR HET OPSLAAN VAN OPGEHAALDE DATA
+            const result = await axios.get(`https://restcountries.com/v2/name/${searchName}`);
+            console.log(result.data);
+            searchResult = result.data[0];
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `searchResult.` vóór hoeft te zetten:
```suggestion
            const { flag, name, subregion, population, capital, currencies, languages } = result.data[0];
```